<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAB_MainWindow</class>
 <widget class="QMainWindow" name="DAB_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>541</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Opérations envoyées à la banque</string>
    </property>
    <widget class="QLineEdit" name="lineEditNumeroCompte_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNumeroCompte_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Numéro de compte</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMontant_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSolde_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Solde</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRetrait_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Retrait</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDepot_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Dépot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnvoi_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoi</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-20</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelEtatConnexion_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat de la connexion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPort_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAdresse_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPort_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAdresse_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnexion_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Message de la banque :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMessageBanque">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>501</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Veuillez  donnez votre numéro de compte</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>DAB_MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
